src/qmapshack/poi/CPoiCategory.h

```diff
@@ -25,18 +25,24 @@
 class CPoiCategory : public QTreeWidgetItem
 {
 public:
-    CPoiCategory(const QString& categoryName, const QString& displayName, Qt::CheckState checkedState, CPoiCategory *parent);
-    CPoiCategory(const QString& displayName, QTreeWidget * parent);
+    CPoiCategory(const QString& categoryName, uint categoryID, Qt::CheckState checkedState, CPoiCategory *parent);
+    CPoiCategory(const QString& categoryName, uint categoryID, QTreeWidget * parent);
 
     const QString& getCategory()
     {
         return categoryName;
     }
 
+    uint getId()
+    {
+        return categoryID;
+    }
+
     Qt::CheckState checkState();
```

Comment:
add a `const` behind each method that does not change the content of the object. If we wouldn't do we couldn't use these methods via const. references.